avaframe/runScripts/runAna3AIMEC.py

```diff
@@ -47,14 +43,15 @@ def runAna3AIMEC(avalancheDir=''):
     cfgSetup = cfg['AIMECSETUP']
     anaMod = cfgSetup['anaMod']
 
-    # Setup input from com1DFA
+    # Setup input from computational module
     inputsDF, resTypeList = dfa2Aimec.mainDfa2Aimec(avalancheDir, anaMod, cfg)
-
     # define reference simulation
-    refSimulation, inputsDF, colorParameter = aimecTools.fetchReferenceSimNo(avalancheDir, inputsDF, anaMod, cfgSetup)
-    pathDict = {'refSimulation': refSimulation, 'compType': ['singleModule', anaMod], 'colorParameter': colorParameter}
+    refSimRowHash, refSimName, inputsDF, colorParameter = aimecTools.fetchReferenceSimNo(avalancheDir, inputsDF, anaMod,
+                                                                                         cfgSetup)
+    pathDict = {'refSimRowHash': refSimRowHash, 'refSimName': refSimName, 'compType': ['singleModule', anaMod],
+                'colorParameter': colorParameter, 'resTypeList': resTypeList}
```

Comment:
shall we consider making this a function?

Reply:
you mean this one line of code? does it make sense?

Reply:
I just think it is not so nice that this has to be added in every run script, or can it be part of another function, but I think you don't have all the inputs available in the others..

Reply:
Not sure I get how you would want to turn that into a function? 
I think this is fine here...